templates/frontOffice/default/account.html

```diff
@@ -60,7 +60,7 @@ <h4 class="panel-title">
                                     {if $CELLPHONE != ""}
                                         <span class="mobile">{$CELLPHONE}</span>
                                     {/if}
-                                    <span class="email">{mailto address="{$EMAIL}" encode="hex"}</span>
+                                    <span class="email">{mailto address=$EMAIL encode="hex"}</span>
```

Comment:
and if for any reason there is a bug we are screwed and we can have a XSS. So please never ever remove the curly bracket, event if you think there is no danger.